app/views/people/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<h1>Task Owner List</h1>
+
+<table>
+  <tr>
+    <th>
+      Name
+    </th>
+    <th>
+      Incomplete Tasks
+    </th>
+  </tr>
+  <tbody>
+    <% @all_people.each do |person| %>
+    <tr>
+      <td>
+        <%= link_to "#{person.name}", "/people/#{person.id}" %>
+      </td>
+      <td>
+        <% if Task.where("person_id = #{person.id}").present? %>
+          <%= Task.where("person_id = #{person.id}").where("completed_date IS NULL ").count %>
```

Comment:
I figured if a Person hasn't even been assigned to any Tasks, they really shouldn't have a count for how many Tasks assigned to them are incomplete.